steal.lic

```diff
@@ -256,7 +256,18 @@ class Steal
 
         drop_item(item) unless keep_item
 
-        move('go portal') if XMLData.room_title == '[[A Junk Yard]]'
+        if XMLData.room_title == '[[A Junk Yard]]'
+		      case bput('look portal', 'You are now able to go through', 'You must get below', 'Please get rid of')
+		      when 'You are now able to go through'
+		        move('go portal')
+		      else
+		        bin_item
```

Comment:
Is there any danger that `@bin_items` will be empty?

Reply:
I believe the only way @bin_items is empty, but a character is in the Junk Room, is if the character's item count is beyond its limit prior to running the ;steal script. I don't think it's advisable or possible to have the script eliminate items from the character beyond those recently stolen items in @bin_items. So they're basically stuck in the Junk Room until items are manually dropped.

Would you suggest the script exiting if @bin_items is empty?

Reply:
I think that makes sense (and `echo` an explanatory message)